PushNotifications.md

```diff
@@ -338,6 +350,18 @@ Do this only if you do not want the device to receive push notifications at all.
     }
 ```
 
+## Debugging applications from launch
+
+You may want to validate or debug situations where your application is terminated, in the background or in the foreground. 
```

Comment:
It's not clear from this sentence whether the validation/debugging happens in the background/foreground, or the terminated application is in the background/foreground. Can we be clearer on this please?

Reply:
I've tried to clarify it, but it might not be enough, as I don't understand the issue you're facing 😅 92e885349aa218633f04fe69333c90e31e0a09aa

There's some pre-requisite knowledge needed here, that there are 3 states I am referring to: terminated, background and foreground.

Reply:
Much clearer, thanks!